### ISSUE TITLE
freehand tool problems

### ISSUE THREAD

**Issue by guiok** [excalidraw/excalidraw#3774]
I'm having problems when using the freehand tool.
It loses line continuity when I am using the pen tablet.
I have attached a video to show you the problem.

![foo](https://user-images.githubusercontent.com/29389316/123781190-52cae280-d8aa-11eb-8dca-96bd0f240403.gif)

**Comment by NayamAmarshe** [comment]
Yes, it's a very annoying issue present on both desktop and mobile.

**Comment by dwelle** [comment]
Known issue tracked here: https://github.com/excalidraw/excalidraw/issues/3564

@steveruizok I know you were working on a fix. Has it been published by any chance yet?
